### ISSUE TITLE
Results  are different from paper when using single domain for training

### ISSUE THREAD

**Issue by haoni0812** [bismex/MetaBIN#8]
Thank you for making your work public, but  when I use market or duke as the training set, the experimental results are always lower than those in paper. I conducted multiple experiments, and each time I got exactly the same experimental results, even including the loss during training. I wanna know what causes this?
Specifically, when I use duke as the training set and market as the test set, the rank1 and map are **66.48%** and **33.82%**, respectively. And in the paper they are **69.2%** and **35.9%** respectively.
Thank you again for your contribution and look forward to your answer!

**Comment by bismex** [comment]
Thanks for the valuable comments. 
Did you use the official config setting (D-resnet.yml)?

**Comment by haoni0812** [comment]
I am sure that I have used  the official config setting (D-resnet.yml) for training. I reproduced the experiment for many times, the results obtained and the loss value during the training process are basically unchanged.
The GPU I use is 1080ti, I don’t know if this will influence the results.

**Comment by bismex** [comment]
Sorry for the confusion. I am sure that all the experimental results recorded in the paper have been actually performed. However, even after getting the final result, I modified the internal code several times, so there may have been a conflict somewhere. Unfortunately, I am currently graduating and doing other research, so it is difficult to find an old version of the code or reproduce the results right now.

Instead, I share the hyper-parameters that I mostly modified in my cross-domain experiments. You may get better results by experimenting with this change once. Please check here (https://github.com/bismex/MetaBIN/blob/master/fastreid/config/defaults.py) for variable description.

In D-resnet.yml,
1) color jittering: INPUT.CJ.ENABLED
2) LR for balancing parameter: META.SOLVER.LR_FACTOR.GATE
3) number of domains in meta-train: SOLVER.MTRAIN.NUM_DOMAIN
4) minibatch: META.DATA.MTRAIN_MINI_BATCH, META.DATA.MTEST_MINI_BATCH, SOLVER.IMS_PER_BATCH
5) number of instances: META.DATA.MTRAIN_NUM_INSTANCE, META.DATA.MTEST_NUM_INSTANCE, DATALOADER.NUM_INSTANCE

And the following variables may also have influenced the results. (Unfortunately I'm not sure)
1) color jittering hyperparameter (fastreid/config/defaults.py -> line 347-353)
2) color jittering package (try to change the color jittering package -> fastreid/data/transforms/build.py/line 60)
3) data loader (fastreid/config/defaults.py -> line 44-46)
4) seed (fastreid/config/defaults.py -> line 110-112)

Also, please check if SOLVER.AMP is applied.

If you get any good results, please reply again. And if I can run the experiment again later or if I get a fix for this, I'll share it again.

**Comment by bismex** [comment]
For reference, the resulting screenshots are all shown here (https://github.com/bismex/MetaBIN/tree/master/logs/Sample).

**Comment by haoni0812** [comment]
Thank you for your patient response, I will try your suggestions and let you know if I get better results

**Comment by meghana-shastri24** [comment]
Hi haoni0812,

Did you get better results? If so, can you please share the modified configuration?

Thanks in advance.
